universal/src/formats/json/impl/accept.hpp

```diff
@@ -132,7 +139,7 @@ bool WriteStartAndEnterValue(Stack<kProcessing>& stack, Handler& handler) {
           const SubRange<MemberIterator>& member_sub_range) mutable -> Value& {
         was_written =
             handler.Key(member_sub_range.begin->name.GetString(),
-                        member_sub_range.begin->name.GetStringLength());
+                        member_sub_range.begin->name.GetStringLength(), true);
```

Comment:
nit: `/*copy=*/ true`
so that one doesn't have to go and look up what it means every time